src/provider/view/components/SettingsView/SettingsView.module.scss

```diff
@@ -43,4 +43,9 @@
             border-bottom: 1px solid var(--color-light);
         }
     }
+
+    .placeholder-text {
```

Comment:
Can the placeholder-text be changed to something more descriptive of what it actually is?

Reply:
Well what would you call it? It's the placeholder text that shows when you have no feeds

Reply:
empty-message/no-feeds-text?

Reply:
👍